examples/README

```diff
@@ -1,5 +1,29 @@
-Sample of OpSim Psf cutout scripts.
+shear_test_experiments
+July 28, 2015
 
-makepsfs is a shell script which calls the necessary python scripts in the bin directory.
+1.  Sample of OpSim Psf cutout scripts.  
```

Comment:
These are from *PhoSim*, not *OpSim* - OpSim is something entirely different (it simulates the cadence observation schedule).